pkg/api/v1/types.go

```diff
@@ -1706,6 +1706,20 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// DaemonEndpoint stores information about a single Daemon endpoint.
+type DaemonEndpoint struct {
+	// Port number of the given endpoint.
+	Port int `json:port`
+}
+
+// NodeDaemonEndpoints lists ports opened by daemons running on the Node.
+type NodeDaemonEndpoints struct {
+	// Endpoint on which Kubelet is listening.
+	KubeletEndpoint DaemonEndpoint `json:"kubeletEndpoint,omitempty"`
+	// Endpoint opened by Kubelet on which cAdvisor proxy is available.
+	CAdvisorProxyEndpoint DaemonEndpoint `json:"cAdvisorProxyEndpoint,omitempty"`
```

Comment:
Good question. I was under the impression that heapster/other monitoring uses it, but it seems that  they use ReadOnly port instead. If this is the case is there any reason for exposing cAdvisor? @dchen1107 @yujuhong